stake-pool/cli/src/main.rs

```diff
@@ -696,10 +852,20 @@ fn command_list(config: &Config, stake_pool_address: &Pubkey) -> CommandResult {
         println!("Validator List: {}", stake_pool.validator_list);
         println!("Manager: {}", stake_pool.manager);
         println!("Staker: {}", stake_pool.staker);
-        println!("Depositor: {}", stake_pool.deposit_authority);
+        println!("Depositor: {}", stake_pool.stake_deposit_authority);
+        println!("SOL Deposit Authority: {}", sol_deposit_authority);
         println!("Withdraw Authority: {}", pool_withdraw_authority);
         println!("Pool Token Mint: {}", stake_pool.pool_mint);
         println!("Fee Account: {}", stake_pool.manager_fee_account);
+        println!("Epoch Fee: {}", stake_pool.fee);
```

Comment:
This is now being printed twice -- could you remove the other print?

Reply:
oh i didn't realize that block was there already, i think we can standardize that if/else block for all the fee types. It provides more info for the user

Reply:
Either way works for me :smile: